platform/linux-generic/pktio/pcap.c

```diff
@@ -294,9 +294,18 @@ static int pcapif_recv_pkt(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 		if (pktio_cls_enabled(pktio_entry)) {
 			odp_packet_t new_pkt;
 
+			packet_parse_reset(pkt_hdr, 1);
+			packet_set_len(pkt_hdr, pkt_len);
```

Comment:
Seems that this set length call is not needed, as alloc and pull_head should have set it correctly.

Reply:
In v4, removed these and a packet_parse_reset() call in sock_mmsg_recv().